scripts/azure-ci-test.sh

```diff
@@ -100,7 +107,7 @@ main() {
   deploy_gatekeeper
   deploy_ratify
 
-  bats -t ./test/bats/azure-test.bats
+  bats -t ./test/bats/azure-test.bats || true
```

Comment:
curious why " true" is needed

Reply:
I found if the bats test failed, it will not execute the next line which uploads the logs.

Reply:
that's for debugging purpose, just forgot to delete it. thanks for catching it.